### PR TITLE
Added row in friends list to add friends

### PR DESCRIPTION
**Pull request by bart-at-mobicage**
The plus or magnifying glass icon alone aren't clear enough for some users. I added a row in the Friends List which opens the AddFriendsVC.

I also removed the top margin of the header image in case of full width headers

### CODE REVIEW

File: src/MCTFriendsTabVC.m [hunk 1]
```diff
@@ -115,19 +115,89 @@ - (void)onMapButtonClicked:(id)sender
...
+                                                 fontSize:20];
+            cell.textLabel.text = text;
+            cell.textLabel.textColor = primaryColor;
+            [self applyStyleToView:cell recursive:YES];
+        }
+        return cell;
+    }
+
+    return [super tableView:tableView
+      cellForRowAtIndexPath:[NSIndexPath indexPathForRow:indexPath.row + 1 inSection:indexPath.section]];
```
Comment (RubenMattan) [inline src/MCTFriendsTabVC.m]: Should be `indexPath.row - 1`

File: src/MCTFriendsTabVC.m [hunk 2]
```diff
@@ -115,19 +115,89 @@ - (void)onMapButtonClicked:(id)sender
...
 
+- (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    T_UI();
+    if (indexPath.section == 0) {
+        if (indexPath.row == 0) {
+            return NO;
+        } else {
+            return [super tableView:tableView
+              canEditRowAtIndexPath:[NSIndexPath indexPathForRow:indexPath.row - 1
```
Comment (RubenMattan) [inline src/MCTFriendsTabVC.m]: Should be `indexPath.row + 1`

Comment (bart-at-mobicage) [reply src/MCTFriendsTabVC.m]: actually not, because eg. row 3 is actually row 2 in the super class.

File: src/MCTFriendsTabVC.m (hunk 1 above)
Comment (bart-at-mobicage) [reply src/MCTFriendsTabVC.m]: oops, indeed

File: src/MCTFriendsTabVC.m (hunk 2 above)
Comment (RubenMattan) [reply src/MCTFriendsTabVC.m]: Add friends is counted as well

**Pull request merged**